src/main/java/it/smartcommunitylab/aac/webauthn/provider/WebAuthnCredentialsService.java

```diff
@@ -0,0 +1,127 @@
+package it.smartcommunitylab.aac.webauthn.provider;
+
+import org.springframework.util.Assert;
+
+import it.smartcommunitylab.aac.SystemKeys;
+import it.smartcommunitylab.aac.common.NoSuchUserException;
+import it.smartcommunitylab.aac.core.base.AbstractProvider;
+import it.smartcommunitylab.aac.core.model.UserCredentials;
+import it.smartcommunitylab.aac.core.provider.CredentialsService;
+import it.smartcommunitylab.aac.webauthn.model.UserWebAuthnCredentials;
+import it.smartcommunitylab.aac.webauthn.persistence.WebAuthnUserAccount;
+import it.smartcommunitylab.aac.webauthn.service.WebAuthnUserAccountService;
+
+public class WebAuthnCredentialsService extends AbstractProvider implements CredentialsService {
```

Comment:
this class needs fixes, remove it for now

Reply:
Removed in c999afbb8dd54c82b7688de3b8b7dfec1bd41502